app/controlers/pages_controller.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class PagesController < ApplicationController
```

Comment:
Looks like maybe this file was an accidental copy/paste?  This is "app/controlers/pages_controller.rb" ("controlers" with one L), duplicates the existing "app/controllers/pages_controller.rb" ("controlers" with two Ls).